cli/tests/integration_tests/rust_tests/crud.rs

```diff
@@ -0,0 +1,451 @@
+use std::collections::HashMap;
+
+use crate::framework::prelude::*;
+
+static PERSON_MODEL: &str = r#"
+    import { ChiselEntity, labels } from "@chiselstrike/api";
+
+    export class Person extends ChiselEntity {
+        first_name: string = "";
+        last_name: string = "";
+        age: number = 0;
+        human: boolean = false;
+        height: number = 1;
+    }
+"#;
+
+static PEOPLE_CRUD: &str = r#"
+    import { Person } from "../models/person.ts";
+    export default Person.crud();
+"#;
+
+lazy_static::lazy_static! {
+    static ref DEJAN: serde_json::Value = json!({
+        "first_name":"Dejan",
+        "last_name":"Mircevski",
+        "age": 42,
+        "human": true,
+        "height": 7
+    });
+}
+
+lazy_static::lazy_static! {
+    static ref GLAUBER: serde_json::Value = json!({
+        "first_name":"Glauber",
+        "last_name":"Costa",
+        "age": 666,
+        "human": true,
+        "height": 10.01
+    });
+}
+
+lazy_static::lazy_static! {
+    static ref HONZA: serde_json::Value = json!({
+        "first_name":"Honza",
+        "last_name":"Spacek",
+        "age": 314,
+        "human": false,
+        "height": 11.11
+    });
+}
+
+lazy_static::lazy_static! {
+    static ref JAN: serde_json::Value = json!({
+        "first_name":"Jan",
+        "last_name":"Plhak",
+        "age": -666,
+        "human": true,
+        "height": 10.02
+    });
+}
+
+lazy_static::lazy_static! {
+    static ref PEKKA: serde_json::Value = json!({
+        "first_name":"Pekka",
+        "last_name":"Heisenberg",
+        "age": 2147483647,
+        "human": false,
+        "height": 12742333
+    });
+}
+
+async fn store_person(chisel: &Chisel, person: &serde_json::Value) -> String {
+    let resp = chisel
+        .post("/dev/people")
+        .json(person)
+        .send()
+        .await
+        .assert_ok()
+        .json();
+    resp["id"].as_str().unwrap().into()
+}
+
+async fn store_all_people(chisel: &Chisel) -> HashMap<String, String> {
+    let mut ids = HashMap::default();
+    ids.insert("Dejan".to_owned(), store_person(chisel, &DEJAN).await);
+    ids.insert("Glauber".to_owned(), store_person(chisel, &GLAUBER).await);
+    ids.insert("Pekka".to_owned(), store_person(chisel, &PEKKA).await);
+    ids.insert("Honza".to_owned(), store_person(chisel, &HONZA).await);
+    ids.insert("Jan".to_owned(), store_person(chisel, &JAN).await);
+    ids
+}
```

Comment:
nit, but you can do that:
```suggestion
async fn store_all_people(chisel: &Chisel) -> HashMap<&'satic str, String> {
    let mut ids = HashMap::default();
    ids.insert("Dejan", store_person(chisel, &DEJAN).await);
    ids.insert("Glauber"), store_person(chisel, &GLAUBER).await);
    ids.insert("Pekka", store_person(chisel, &PEKKA).await);
    ids.insert("Honza", store_person(chisel, &HONZA).await);
    ids.insert("Jan", store_person(chisel, &JAN).await);
    ids
}
```
p.s: where is marin?